client/components/NavigationBar/NavigationBar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export class NavigationBar extends React.Component {
+  render() {
+    return(
+      <nav className="cyan darken-3">
+        <div className="nav-wrapper">
+          <a href="#" className="brand-logo">&nbsp; OneLaptop</a>
```

Comment:
I notice you're using &nbsp; and a space in the brand logo. Please refactor it and use padding instead for consistency.